firmware/quadruna/BMS/src/app/states/app_allStates.c

```diff
@@ -29,8 +29,9 @@ uint32_t    iso_spi_state_counter = 0;
 
 void app_allStates_runOnTick1Hz(void)
 {
-    bool charger_is_connected = io_charger_isConnected();
-    app_canTx_BMS_ChargerConnected_set(charger_is_connected);
+    bool charger_is_connected = app_canRx_BRUSA_IsConnected_get();
+    // Feel like this wouldn't be needed if the BRUSA broadcasts over CAN
```

Comment:
I tend to agree. The behaviour we want is to remain in Init State if the BMS detects it is plugged in via the charging harness outside the car (Check for consistent missing heartbeats perhaps?), then transition to charging state if the debug message is sent

Reply:
Also, maybe in case we accidentally go to drive state, have a condition to return to init if the charger is connected? IDK might be overkill